mllib/src/main/scala/org/apache/spark/ml/feature/StringIndexer.scala

```diff
@@ -66,6 +68,9 @@ private[feature] trait StringIndexerBase extends Params with HasHandleInvalid wi
    *   - 'alphabetAsc': ascending alphabetical order
    * Default is 'frequencyDesc'.
    *
+   * Note: In case of equal frequency when under frequencyDesc/Asc, the strings are further sorted
```

Comment:
minor nit: "further sorted alphabetically." could read better than "by alphabet."

Reply:
Good for me.